docker/prover-service.Dockerfile

```diff
@@ -24,15 +24,11 @@ COPY contracts/ /work/contracts/
 
 # copy the binaries
 COPY target/$TARGETARCH/release/state-prover /usr/local/bin/state-prover
-COPY target/$TARGETARCH/release/gen-demo-genesis /usr/local/bin/gen-demo-genesis
-COPY scripts/launch-prover-service /usr/local/bin/launch-prover-service
 RUN chmod +x /usr/local/bin/state-prover
-RUN chmod +x /usr/local/bin/launch-prover-service
-RUN chmod +x /usr/local/bin/gen-demo-genesis
 
 # When running as a Docker service, we always want a healthcheck endpoint, so set a default for the
 # port that the HTTP server will run on. This can be overridden in any given deployment environment.
 ENV ESPRESSO_COMMITMENT_TASK_PORT=80
 HEALTHCHECK --interval=1s --timeout=1s --retries=100 CMD curl --fail http://localhost:${ESPRESSO_COMMITMENT_TASK_PORT}/healthcheck || exit 1
 
-CMD [ "launch-prover-service" ]
+CMD [ "state-prover", "-d" ]
```

Comment:
Rust and foundry is no longer needed in this docker.

Reply:
Good point

Reply:
Removed in [6d5f487](https://github.com/EspressoSystems/espresso-sequencer/pull/1262/commits/6d5f48768aa02fbfa2523e3c8c81719dc5549961)